solr/src/test/scala/akka/stream/alpakka/solr/SolrSpec.scala

```diff
@@ -67,18 +68,20 @@ class SolrSpec extends WordSpecLike with Matchers with BeforeAndAfterAll {
         .map { tuple: Tuple =>
           val book: Book = tupleToBook(tuple)
           val doc: SolrInputDocument = bookToDoc(book)
-          IncomingMessage(doc)
+          Seq(IncomingUpdateMessage(doc))
         }
         .runWith(
-          SolrSink.document(
+          SolrSink.documents(
             collection = "collection2",
             settings = SolrUpdateSettings(commitWithin = 5)
-          )
+          )(cluster.getSolrClient)
```

Comment:
Why are you passing the Solr client explicitly?

Reply:
Because the implicit was another solr client, and we do not have to instantiate multiple clients. We could instantiate an implicit lazily with this instance.